src/services/api/pages.js

```diff
@@ -3,21 +3,23 @@ import { SITES, PAGES } from 'common/constants'
 
```

Comment:
@gmcauliffe I tried really hard not to add any incorrect {} back into these, but could you please double check as well? They are hard to notice.